src/stories/HeatmapVis.stories.tsx

```diff
@@ -19,7 +19,23 @@ const logSafeDomain = getDomain(dataValues, ScaleType.Log);
 const alphaArray = ndarray(dataValues.map((x) => Math.abs(x)));
 const alphaDomain = getDomain(alphaArray);
 
-const Template: Story<HeatmapVisProps> = (args) => <HeatmapVis {...args} />;
+const Template: Story<
+  Omit<HeatmapVisProps, 'scaleType'> & {
+    scaleType: ScaleType;
+    gammaExponent: number;
+  }
+> = (args) => {
+  const { scaleType, gammaExponent, ...otherArgs } = args;
+  const props = {
+    scaleType:
+      scaleType === ScaleType.Gamma
+        ? ([ScaleType.Gamma, gammaExponent] as [ScaleType.Gamma, number])
```

Comment:
`as const` maybe ?

Reply:
Get casted as `readonly`. 
Then TS complains that `readonly [ScaleType.Gamma, number]` cannot be set to `[ScaleType.Gamma, number]` when doing `<HeatmapVis {...props} />` -____-